packages/block-editor/README.md

```diff
@@ -651,6 +651,26 @@ _Related_
 
 -   <https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/line-height-control/README.md>
 
+### LinkControl
+
+Renders a link control. A link control is a controlled input which maintains a value associated with a link (HTML anchor element) and relevant settings for how that link is expected to behave.
+
+_Parameters_
+
+-   _props_ `WPLinkControlProps`: Component props.
```

Comment:
I think I would be in favor of stabilizing LinkControl but not its internal components. I think these are implementation details that shouldn't be exposed publicly.

Reply:
Yes I agree. I don't feel they should ever have been exposed really even as `__experimental`.

Reply:
Done  👍